### ISSUE TITLE
Telegram bot not functional

### ISSUE THREAD

**Issue by ianlopez1115** [Zverik/osmstreak#80]
Telegram bot is unresponsive or not functioning since September 23, 2024. Tried various commands then blocking/unblocking+restarting the bot, still doesn't work. Tried again a few minutes ago (see attached screencast), still not working.

https://github.com/user-attachments/assets/6ad33d93-c848-4182-ab9a-d44e8deb959a

**Comment by Zverik** [comment]
Yes, I am sorry for that. I have finally moved OSM Streak to a new server, so it works properly now with OAuth authentification. But I did not update the bot code yet.
